requirements-dev.txt

```diff
@@ -14,6 +14,10 @@ aniso8601==9.0.1
     #   flask-restx
 asserts==0.11.1
     # via -r requirements-dev.in
+async-timeout==4.0.2
+    # via
+    #   -r requirements.txt
+    #   redis
```

Comment:
Don't think we're using redis anymore after this refactoring? If so please can we remove it from `requirements.in` and re-generate the `requirements.txt` file?